problem_builder/translations/config.yaml

```diff
@@ -3,15 +3,15 @@
 locales:
     - en  # English - Source Language
     - ar  # Arabic
-    - es_419  # Spanish
-    - pt_BR  # Portuguese
-    - zh_CN  # Chinese
+    - es-419  # Spanish
+    - pt-br  # Portuguese
+    - zh-cn  # Chinese
     - fr  # French
-    - fr_CA  # French (Canada)
-    - de_DE  # German
-    - ja_JP  # Japanese
-    - pl_PL # Polish
-    - ko_KR #Korean
+    - fr-ca  # French (Canada)
+    - de-de  # German
+    - ja-jp  # Japanese
+    - pl-pl # Polish
+    - ko-kr #Korean
     # Add languages here as needed
```

Comment:
I think the original names were what are used for edx-platform so we should stay consistent with that: https://github.com/edx/edx-platform/blob/0935b5c51fa4115caa6d6d98b0ef65fea4fcdec1/conf/locale/config.yaml#L9-L91

Reply:
@xitij2000 Django3.2 refused to start with the original names. It would throw [this error](https://github.com/open-craft/problem-builder/pull/333).

I looked at the [LANGUAGES setting in edx-platform](https://github.com/edx/edx-platform/blob/0935b5c51fa4115caa6d6d98b0ef65fea4fcdec1/lms/envs/common.py#L1714) and they do use the django3.2 compatible names, so I thought it was ok to change this, but I don't really know how translations work with this XBlock.

I didn't test whether translations are working - I don't know if that's possible to do inside xblock-sdk.

Reply:
I don't think it's possible with xblock-sdk, but I've done it a lot, so I'll have a quick look with one of the affected languages.

Reply:
es-419:
![Screenshot 2021-10-01 at 09-40-54 Subsection Section Problem Builder Test edX](https://user-images.githubusercontent.com/118837/135599515-34b5c982-31a3-4237-bb71-ab211363ac8f.png)

ja-jp:
![Screenshot 2021-10-01 at 09-38-58 Subsection Section Problem Builder Test edX](https://user-images.githubusercontent.com/118837/135599563-968af256-3309-4557-b9c5-9e89d2e2752a.png)


Seems to be working :-)

Reply:
Great, thanks a lot for checking :)